core/src/main/scala/io/finch/Accept.scala

```diff
@@ -0,0 +1,22 @@
+package io.finch
+
+import cats.Eq
+import javax.activation.MimeType
+import scala.util.control.NonFatal
+
+case class Accept(primary: String, subtype: String)
+
+object Accept {
+
+  def apply(s: String): Option[Accept] = try {
```

Comment:
Can we rename it to `fromString` (otherwise it looks like a case class constructor, which is a total function).